package-res/ccc/core/base/panel/panel/panel.js

```diff
@@ -576,31 +601,33 @@ def
                     childKeyArgs.canChange = remTimes > 0;
 
                     child.layout(new pvc_Size(remSize), childKeyArgs);
+             
                     if(child.isVisible) {
                         resized = checkChildResize.call(this, child, canResize);
                         if(resized) return false; // stop
-
-                        var requestPaddings = child._layoutInfo.requestPaddings;
-                        if(checkPaddingsChanged(paddings, requestPaddings)) {
-                            paddings = requestPaddings;
-
-                            // Child wants to repeat its layout with != paddings
-                            if(remTimes > 0) {
-                                paddings = new pvc_Sides(paddings);
-                                if(useLog) this.log("Child requested paddings change: " + def.describe(paddings));
-                                return true; // again
+                        
```

Comment:
Indentation got a bit messed up here.
Please remove the added and commented preserveLayout test.
Was it really intended to move the positionChild to outside of the child.isVisible "if"?